includes/User.php

```diff
@@ -246,6 +247,11 @@ class User {
 	static $idCacheByName = array();
 
 	/**
+		* @var string the service auth token (currently helios); should NEVER be cached
+	 */
+	private $globalAuthToken = null;
```

Comment:
Yah that makes sense, but I remember a case when I tried to have 

```php
private $preferenceService

private function getPreferenceService() {
  return $this->preferenceService;
}
```

and I got an error about trying to serialize a database connection, which I think I tracked down to some caching going on somewhere.